habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/ProcessExecutor.java

```diff
@@ -187,7 +190,15 @@ public final void flush() {
 
```

Comment:
Before we were only throwing all the logs out on a failure in a logger.error message. This is an attempt to output the Python logs with the correct logger level. Open to discussions about a more robust solution.